atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/api/cache/CacheMetrics.java

```diff
@@ -87,6 +87,10 @@ public void decreaseCacheSize(long removed) {
         cacheSize.dec(removed);
     }
 
+    public void resetCacheSize() {
+        cacheSize.dec(cacheSize.getCount());
+    }
```

Comment:
why there isn't a reset method is beyond me

Reply:
Probably they expect you to throw it away and create a new counter?